src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs

```diff
@@ -617,6 +620,35 @@ private BuildEventArgs ReadBuildFinishedEventArgs()
             return e;
         }
 
+        private BuildEventArgs ReadBuildSubmissionStartedEventArgs()
+        {
+            var fields = ReadBuildEventArgsFields();
+
+            IDictionary<string, string>? globalProperties = null;
+            // See ReadProjectEvaluationFinishedEventArgs for details on why we always store global properties in newer version.
+            if (_fileFormatVersion >= BinaryLogger.ForwardCompatibilityMinimalVersion ||
```

Comment:
The `_fileFormatVersion >= BinaryLogger.ForwardCompatibilityMinimalVersion ||` should not be needed - as this event is newly introduced - it won't be written in older version of logs